src/js/_enqueues/wp/customize/preview.js

```diff
@@ -673,7 +673,7 @@
 				css += 'background-attachment: ' + settings.attachment() + ';';
 			}
 
-			$( '#custom-background-css' ).text( 'body.custom-background { ' + css + ' }' );
+			$( '#custom-background-inline-css' ).text( 'body.custom-background { ' + css + ' }' );
```

Comment:
Changing this ID will also be problematic, especially for AMP: https://wpdirectory.net/search/01H9VJD4B8QSMEGQJ13SPNNE8B

Reply:
See https://github.com/WordPress/wordpress-develop/pull/4824/files#r1321335752